models/router.go

```diff
@@ -0,0 +1,63 @@
+package models
```

Comment:
The router isn't a model. It should be part of the main package unless there's a specific reason to split it out?

Reply:
The router is used in the tests, if i have it in the main package this causes issues. Unless there's a way to use the router function from the main package in tests?